src/dto.py

```diff
@@ -7,6 +22,7 @@ def validateDto(message):
               message['harena-log-stream-version'] = str(enums.StreamVersionSpecialCodes.NOT_IDENTIFIED.value)
         
         #TODO: create an exception handler with custom exceptions
+       
```

Comment:
Even when the DTO is not as marshmallow expects it to be. It would be interesting if it was sent on kafka.